### ISSUE TITLE
Gekko encountered an error and can't continue [binance.js] Response code 400

### ISSUE THREAD

**Issue by AndreiGhitan** [askmike/gekko#2037]
I am getting the following error with the livetrader on binance:

RECEIVED ERROR IN GEKKO 0101215627964186

Gekko encountered an error and can't continue

Gekko encountered an error and can't continue

Error:

[binance.js] Response code 400

Meta debug info:

Gekko version: v0.5.12
Nodejs version: v9.8.0

Since I started it, it did one trade for me meaning it bought the asset with ETH, and minutes later this error popped up and I tried everything, restarted my machine and also followed the instructions to update binance to 1.2.2. Every time now when I strt the live trader it gives me this error.

Can you guys help me get this fixed please?

**Comment by bbkilboz** [comment]
Can you check your system clock? 
There was a difference between my system clock and binance server clock, on my instance, and when i synced my system clock with NTP it started working again.

**Comment by AndreiGhitan** [comment]
I have checked it and it is set on automatic. I am running it on Mac OSX High Sierra.

How do I sync it with NTP on mac?

**Comment by bbkilboz** [comment]
If it's automatic then it should be syncing with a NTP server of Apple. 
You can try to change its server to another Apple server or 0.pool.ntp.org but I doubt that'll work. 
Hope you find a solution soon

**Comment by AndreiGhitan** [comment]
I followed some of the tips posted in other threads for the solution but it still gives me this error, would be really awesome if somebody could help me get this sorted

**Comment by askmike** [comment]
In what region (in the world) are you based? And do you have a fast internet connection?

**Comment by AndreiGhitan** [comment]
In Romania, and yes my internet connection is fast and stable.

**Comment by JoeTeglasi** [comment]
I’m also having this problem

**Comment by JoeTeglasi** [comment]
Actually nvm. My issue was because I changed my binance orders from LIMIT to MARKET, and the api didn’t like that I left the other settings unchanged. So for me the fix was to comment-out the time in force and price

**Comment by AndreiGhitan** [comment]
I will try running it in windows a bit later and see if it does the same. It kills me though cause I found a killer strategy and can't use it.

Only thing it does is that it buys the asset and never switches back. It just pops that error. Would love to get this running as soon as possible since I really like this bot :)

**Comment by AndreiGhitan** [comment]
it’s even worse on windows, as soon as I launched the live trader it popped the error :(

**Comment by askmike** [comment]
Sorry about that, Gekko was not properly printing the actual error, please download the latest develop branch and try again :)

**Comment by hoppy1969** [comment]
I'm getting the same error.  I just downloaded the gekko-develop today.  Everything seemed to download fine, but if I try to import data or start a new gekko I get the following error: Gekko encountered an error and can't continue.  In my command window it looks like this:
Gekko 308934527360494 started
  --> POST /api/startGekko 200 11ms 148b
RECEIVED ERROR IN GEKKO 308934527360494


Gekko encountered an error and can't continue


Gekko encountered an error and can't continue

Error:

{}

Meta debug info:

Gekko version: v0.5.14
Nodejs version: v8.11.1

I saw some people talking about time, but mine matches up to the binance clock which is the local time I'm working on here.   What am I missing?

I switched browsers to Firefox and this seems to have fixed the issue.
v/r
Hoppy
